### ISSUE TITLE
Preopen sidenav-collapsibles

### ISSUE THREAD

**Issue by renatop7** [sherweb/ng2-materialize#318]
Is there a way to preopen sidenav-collapsibles like we do with normal collapsibles?

**Comment by scote** [comment]
Yes we should be able to use the same feature as collapsible component. But we did not have add all feature to sidenav collapsible yet.
